lib/bindings/HTTPBindings.js

```diff
@@ -166,10 +166,13 @@ function start(config, callback) {
 }
 
 function stop(callback) {
-    logger.info(context, 'Stopping Ultralight HTTP Binding');
+    logger.info(context, 'Stopping Ultralight HTTP Binding: ' + (new Date()));
```

Comment:
What's the reason to include the current date and time in the log message taking into consideration the `logops`module already logs it in the `time` property? :)